report-step-result/print-file/action.yml

```diff
@@ -21,16 +21,41 @@ inputs:
     description: 'The action should exit with an error status if the specified report file can not be found'
     required: false
     default: 'true'
+  quiet:
+    description: 'Whether to print the report to the console'
+    required: false
+    default: 'true'
 runs:
   using: composite
   steps:
-    - name: Append to step summary
-      run: ${{ github.action_path }}/append-to-step-summary.sh
+    - name: Check input file exists
+      id: check-file-exists
+      if: ${{ inputs.file-path != inputs.output-file-path }}
       shell: bash
       env:
-        FILE_PATH: ${{ inputs.file-path }}
-        OUT_FILE: ${{ inputs.output-file-path }}
+        FILE: ${{ inputs.file-path }}
+        FAIL_IF_FILE_MISSING: ${{ inputs.fail-if-report-missing == 'true' }}
+      run: |
```

Comment:
can we put this inline bash script in script file at /script and use here. It will make it re-usable and testable as well.
i.e `REPORT_FILE_CHECK: ${{ github.action_path }}/../../scripts/report-step-result/check-report-file.sh`

Reply:
We could but this functionality is not needed anywhere else. It's specific to this action only, so I don't think it needs to be put in a separate script.